### DISCUSSION TITLE
How do you remove the contributors section from changelogs?

### DISCUSSION THREAD

**Discussion by Farioko** [jreleaser/jreleaser#696] [Q&A] [answered]
Hi everyone,

I would like to remove the contributors section from my changelogs. The documentation states the following: 

"Contributor names are added to the formatted changelog by default. You may choose to provide a custom format or disable this behavior completely." - https://jreleaser.org/guide/latest/configuration/release/changelog.html#_contributors

However, I am unable to find an explanation on how to disable this behavior. Could someone explain what I need to add to my Maven pom.xml?

**Comment by aalmiray** [comment, marked as answer]
The `contributors` section  has an `enabled` property that you can set to `false`. This property is set to `true` by default.

```xml
<jreleaser>
  <release>
    <github>
      <changelog>
        <contributors>
          <enabled>false</enabled>
        <contributors>
      <changelog>
    </github>
  </release>
</jreleaser>
```

**Comment by Farioko** [comment]
Thank you for your quick reply! This solved my issue. 🙂
